### PR TITLE
E2E Test scripts

### PR DESCRIPTION
**Pull request by melanierogan**
## Who is this for?

For any engineers who want to run through all of our unit and playwright tests before merging their PRs.

## What is it doing for them?

There is now one command that will run all tests `yarn e2e`

### CODE REVIEW

**Comment by rcantin-w** [comment]
This makes me happy, thanks sm for looking into it 😄  

Question; the playwright tests are being run against staging, right, so more for a post PR merge test?

File: package.json [hunk 1]
```diff
@@ -10,7 +10,12 @@
     "content": "yarn workspace @weco/content run dev",
     "build-webapps": "echo '@weco/catalogue @weco/content @weco/identity' | xargs -n 1 | xargs -I% yarn workspace % run build",
     "clean": "rimraf '**/.next **/lib'",
-    "clean-deps": "rimraf '**/node_modules'"
+    "clean-deps": "rimraf '**/node_modules'",
+    "catalogue:test": "echo \"⭕️ Running catalogue tests...\" && cd catalogue/webapp && NODE_ENV=test jest --no-cache && echo \"✅️ catalogue tests finished\"",
```
Comment (davidpmccormick) [inline package.json:14]: How would you feel about having `test:` first? e.g. `test:catalogue`, `test:content`, etc. – feels a bit more natural to me, not sure what others think

File: package.json [hunk 2]
```diff
@@ -10,7 +10,12 @@
...
+    "content:test": "echo \"⭕️ Running content tests...\" && cd content/webapp && NODE_ENV=test jest --no-cache && echo \"✅️ content tests finished\"",
+    "playwright:test": "echo \"⭕️ Running playwright tests...\" && cd playwright && PLAYWRIGHT_BASE_URL=https://www-stage.wellcomecollection.org yarn test && echo \"✅️ playwright tests finished\"",
+    "playwrightMobile:test": "echo \"⭕️ Running playwright mobile tests...\" && cd playwright && PLAYWRIGHT_BASE_URL=https://www-stage.wellcomecollection.org NODE_ENV=test platform=mobile yarn test && echo \"✅️ playwright mobile tests finished\"",
```
Comment (davidpmccormick) [inline package.json:17]: …and I'd probably want to call this `test:playwright:mobile`

**Review by davidpmccormick — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (melanierogan) [reply package.json:14]: that does feel more natural, I'll go with that

File: package.json (hunk 2 above)
Comment (melanierogan) [reply package.json:17]: nice

**Comment by melanierogan** [comment]
> Question; the playwright tests are being run against staging, right, so more for a post PR merge test?

Ah yes indeed. So I guess it is more of a post PR merge to staging workflow...should I adjust the script name from e2e to better capture that? `e2e-stage` maybe?

**Comment by rcantin-w** [comment]
> > Question; the playwright tests are being run against staging, right, so more for a post PR merge test?
> 
> Ah yes indeed. So I guess it is more of a post PR merge to staging workflow...should I adjust the script name from e2e to better capture that? `e2e-stage` maybe?

Maybe! The other two are still useful for local testing, maybe they could have their own command (and do we want to add `common` tests as well?)

**Review by jamieparkinson — APPROVED** [review]
:tada:

**Comment by rcantin-w** [comment]
Had to fix some tests from `identity` this morning, should we add them?

**Review by rcantin-w — APPROVED** [review]
💕 🎉

**Pull request merged**